frontend/ui/widget/checkbutton.lua

```diff
@@ -64,10 +64,11 @@ function CheckButton:initCheckButton(checked)
         width = self.max_width - self._checkmark.dimen.w,
         fgcolor = self.enabled and Blitbuffer.COLOR_BLACK or Blitbuffer.COLOR_DARK_GRAY,
     }
+    local textbox_shift = self._checkmark.baseline - self._textwidget:getBaseline()
     self._verticalgroup = VerticalGroup:new{
         align = "left",
         VerticalSpan:new{
-            width = self.face.size * 0.18,
+            width = textbox_shift,
         },
         self._textwidget,
```

Comment:
Dunno if this textbox_shift can be negative - and if VerticalSpan behaves with a negative width - but it if can, and it is, you may need to put the other checkmark in a verticalgroup and have the vertical space before it.
(Sometimes, if there's no need to access the textwidget after, it's easier to just replace/rename it with `self._textwidget = VerticalGroup:new{ ... self._textwidget`)

Reply:
I've checked all 6 modules with Checkbuttons, all okay.
I cannot find the combination to make textbox_shift negative with current your baseline tricks.

Reply:
OK, may be just add for safety a `if textbox_shift > 0 then` to avoid useless verticalgroup if 0 and undefined behaviour if negative :)
Or just `if textbox_shift <= 0 then textbox_shift = 0 end` if you want to keep `self._verticalgroup` and not use my trick of replacing in-place `self._textwidget`.